werkzeug/datastructures.py

```diff
@@ -2559,12 +2559,12 @@ def mimetype(self):
         """Like :attr:`content_type` but without parameters (eg, without
         charset, type etc.).  For example if the content
         type is ``text/html; charset=utf-8`` the mimetype would be
-        ``'text/html'``.
+        ``'text/html'``. Also, mimetype should be always lowercase.
```

Comment:
This is correct, but in this case it's more appropriate to say that this property **will** always be lowercase.